web/server/web_client.c

```diff
@@ -1035,13 +1034,15 @@ static inline HTTP_VALIDATION http_request_validate(struct web_client *w) {
                 // a valid complete HTTP request found
 
                 *ue = '\0';
+                //This is to avoid crash in line
+                w->url_search_path = NULL;
```

Comment:
This does not solve the problem. Now it will crash in line 1053 with NULL pointer dereference.